actions/merger/README.md

```diff
@@ -0,0 +1,53 @@
+# Merger Action
+
+
+### Overview
+
+Commits and merges code changed in earlier steps of workflow
+
+### Example Usage
+
+```
+    - name: 'Checkout repo'
+        uses: actions/checkout@v2
+        with:
+          repository: 'broadinstitute/datarepo-helm-definitions'
+          token: ${{ secrets.BROADBOT_TOKEN }}
+          path: datarepo-helm-definitions
+    - name: "Make change"
+        run: <make some change to datarepo-helm-definitions repo here>
+    - name: "Merge using new merge action"
+        uses: broadinstitute/datarepo-actions/actions/merger@0.37.0
+        env:
+          COMMIT_MESSAGE: "Datarepo Integration mutli chart version update"
```

Comment:
How do these `env` variables align with the inputs below? E.g. this is `COMMIT_MESSAGE` but below there is `commit_msg`. I am also confused about using env vars vs inputs; I would expect to see task inputs used here instead of env vars. But maybe this is a usual convention for how github actions are written and used.

Reply:
Great point - I was conflating inputs and environment variables. I've changed it to just use environment variables for now because it seems much more straightforward to deal with in the code.